migrate_nidirect_node/migrate_nidirect_node_article/src/EventSubscriber/PostMigrationSubscriber.php

```diff
@@ -1,121 +0,0 @@
-<?php
-
-namespace Drupal\migrate_nidirect_node_article\EventSubscriber;
-
-use Drupal\migrate\Event\MigrateEvents;
-use Drupal\migrate\Event\MigrateImportEvent;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
-use Drupal\Core\Logger\LoggerChannelFactory;
-
-/**
- * Class PostMigrationSubscriber.
- *
- * Post Migrate processes.
- */
-class PostMigrationSubscriber implements EventSubscriberInterface {
-
-  /**
-   * Drupal\Core\Entity\EntityTypeManagerInterface definition.
-   *
-   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
-   */
-  protected $entityTypeManager;
-
-  /**
-   * Drupal\Core\Logger\LoggerChannelFactory definition.
-   *
-   * @var \Drupal\Core\Logger\LoggerChannelFactory
-   */
-  protected $logger;
-
-  /**
-   * Array of nodes that should be removed from the site.
-   *
-   * @var array
-   */
-  protected $nodes;
-
-  /**
-   * PostMigrationSubscriber constructor.
-   */
-  public function __construct(EntityTypeManagerInterface $entityTypeManager, LoggerChannelFactory $logger) {
-    $this->entityTypeManager = $entityTypeManager;
-    $this->logger = $logger->get('migrate_nidirect_node_article');
-
-    // Array of nodes imported from the Money Advice Service (MAS). These will
-    // be deleted once we have migrated the article revisions as they will be
-    // periodically imported/updated.
-    $this->nodes = [
```

Comment:
Now we know we don't need to keep the original nids, we can safely ignore these rather than process + delete after.